src/data/string/ustring.h

```diff
@@ -75,72 +76,121 @@ uchar string_to_uchar(const char* p);
 uchar string_to_uchar(const std::string& s);
 std::string uchar_to_string(uchar uc);
 
+class fallback_base {
+public:
+  virtual ~fallback_base() {
+  }
+
+  virtual uchar fallback(const std::string& bytes,
+			 const char* hint) = 0;
+};
+
+class exception_fallback : public fallback_base {
+public:
+  virtual ~exception_fallback() {
+  }
+
+  virtual uchar fallback(const std::string& bytes,
+			 const char* hint) {
+    throw std::invalid_argument(hint);
+    return 0x0000;
```

Comment:
Unreachable code.
Since this `0x0000` will never be used, better to delete this line?